lib/zookeeper.js

```diff
@@ -30,7 +36,7 @@ var Zookeeper = function (connectionString, options) {
 util.inherits(Zookeeper, EventEmiter);
 
 Zookeeper.prototype.registerConsumer = function (groupId, consumerId, payloads, cb) {
-    var path = '/consumers/' + groupId;
+    var path = this.pathPrefix+'/consumers/' + groupId;

```

Comment:
A nit here but could we keep the spacing before and after the `+` operator consistent with the previous code? In the future we will need to add JSCS to help enforce this.